llvm/tools/obj2yaml/xcoff2yaml.cpp

```diff
@@ -37,7 +37,7 @@ class XCOFFDumper {
   Error dumpAuxSyms(XCOFFYAML::Symbol &Sym, const XCOFFSymbolRef &SymbolEntRef);
   void dumpFuncAuxSym(XCOFFYAML::Symbol &Sym, const uintptr_t AuxAddress);
   void dumpExpAuxSym(XCOFFYAML::Symbol &Sym, const uintptr_t AuxAddress);
-  void dumpCscetAuxSym(XCOFFYAML::Symbol &Sym,
+  void dumpCsectAuxSym(XCOFFYAML::Symbol &Sym,
```

Comment:
Thanks for correcting typo. :)

Reply:
This should be fixed separately. No need for a review though.

Reply:
I can use a separate PR.  But in general, I think fixing minor typos along with functional changes makes life easier for everyone.  And the typo is related to the functional change.

Reply:
I couldn't find an exact reference from the community guidelines, but it's strongly related to this part of the "How to submit a patch" section describing what a patch must be:
> be an isolated change. Independent changes should be submitted as separate patches as this makes reviewing easier.

The norm is to do the nfc fix immediately before landing the real patch.